apps/cookbook/src/app/showcase/list-showcase/list-showcase.component.ts

```diff
@@ -30,4 +31,42 @@ export class ListShowcaseComponent {
 
   sectionHeaderExampleHtml: string =
     '<GridLayout *kirbyListSectionHeader="let section" ios:height="50">...</GridLayout>';
+  properties: ApiDescriptionProperty[] = [
+    {
+      name: 'showDivider',
+      description: 'Determines if dividers (bottom-border on list items) should be shown or not',
+      defaultValue: 'true',
+      type: ['boolean'],
+    },
+    {
+      name: 'itemSelect',
+      description: 'Emitting event when an item is selected',
+      defaultValue: 'null',
+      type: ['any'],
+    },
+    {
+      name: 'getItemColor',
+      description: 'Will give every other item in the list a ThemeColor',
+      defaultValue: 'null',
+      type: ['ThemeColor'],
+    },
+    {
+      name: 'getSectionName',
+      description: 'Callback to determine name of section. Sections will be ordered alphabetically',
+      defaultValue: 'null',
+      type: ['string'],
+    },
+    {
+      name: 'getStandAloneByProperty',
+      description: 'Property name to decide which items should be stand alone',
```

Comment:
It's basically his description from an old issue.

Reply:
Top notch :)

Reply:
Maybe it would be more clear with a description like this: A boolean property on the item that decides if the item should stand alone. Fx. if the item is `{name: 'myItem',  suspiciousActivity: true }`, then `getStandAloneByProperty ` would be `suspiciousActivity`.

Reply:
I changed over to your new description Mark :)